### PR TITLE
[ADD] New module pos_transfer_account;

### PR DESCRIPTION
**Pull request by legalsylvain**
This module was written to extend the functionality of odoo Point Of Sale about transfer account.

This module is usefull only in a multi Point of Sale context of multi company context.

With this module, it is now allowed to define for each PoS config a transfer account.

If this account is set, it will be used when user realizes "Put Money In" or "Take Money Out" operation, instead of the default one set in 'Invoicing' / 'Configuration' / 'Setting' Section. ("Inter-Banks Transfer Account" field)
Review welcome.

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hi @legalsylvain , could you please review travis? and where is runbot? I can't test it

**Comment by rafaelbn** [comment]
Hi @legalsylvain , could you please rebase? Now runbot is fixed I would like to test it. Thanks!

**Comment by legalsylvain** [comment]
@rafaelbn : done.

**Comment by legalsylvain** [comment]
@rafaelbn, any news ?

**Comment by rafaelbn** [comment]
Rebuilding for testing!

**Review by rafaelbn — APPROVED** [review]
Thanks! And sorry by late answer! :+1:

**Comment by rafaelbn** [comment]
Please @chienandalu review this PR, thanks! Any doubt please let me know.

File: pos_transfer_account/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Point of Sale - Transfer Account',
+    'version': '9.0.1.0.0',
+    'category': 'Point Of Sale',
+    'author': 'La Louve, Odoo Community Association (OCA)',
+    'website': 'http://www.lalouve.net',
```
Comment (chienandalu) [inline pos_transfer_account/__openerp__.py:11]: https://github.com/OCA/pos

File: pos_transfer_account/models/cash_box_in.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+from openerp import api
+
+from openerp.addons.account.wizard.pos_box import CashBox
+
+
+class CashBoxIn(CashBox):
+    _inherit = 'cash.box.in'
+
+    @api.one
```
Comment (chienandalu) [inline pos_transfer_account/models/cash_box_in.py:15]: Why don't you use `@api.multi`?

**Review by chienandalu — COMMENTED** [review]
Looks good overall. Just few comments

**Comment by legalsylvain** [comment]
Hi. Sorry, I'll not work anymore on this module. But feel free to commit on this PR and merge this PR. 
kind regards.

**Comment by rafaelbn** [comment]
@chienandalu let's merge as is, are you agree?

**Comment by chienandalu** [comment]
Well, ok, although it's going to decrease branch coverage...

**Pull request merged**